README.md

```diff
@@ -1,13 +1,13 @@
 # Linear models (lm's) and generalized linear models (glm's) in Julia
 
 [![Travis](https://travis-ci.org/JuliaStats/GLM.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/GLM.jl)
-[![GLM](http://pkg.julialang.org/badges/GLM_0.4.svg)](http://pkg.julialang.org/?pkg=GLM)
+[![Coverage Status](https://coveralls.io/repos/github/JuliaStats/GLM.jl/badge.svg?branch=master)](https://coveralls.io/github/JuliaStats/GLM.jl?branch=master)
 [![GLM](http://pkg.julialang.org/badges/GLM_0.5.svg)](http://pkg.julialang.org/?pkg=GLM)
+[![GLM](http://pkg.julialang.org/badges/GLM_0.6.svg)](http://pkg.julialang.org/?pkg=GLM)
 
 ## Older versions
 
 This documentation applies to the current version of GLM.jl, which requires Julia 0.5 or later.
```

Comment:
Isn't is a safe assumption that any documentation refers to the most recent version? Seems to me that this older versions section could be removed altogether.